client/landing/gutenboarding/onboarding-block/domains/index.tsx

```diff
@@ -75,6 +81,7 @@ const DomainsStep: React.FunctionComponent< Props > = ( { isModal } ) => {
 			</div>
 			<ActionButtons>
 				<BackButton onClick={ handleBack } />
+				{ domain ? <NextButton onClick={ handleNext } /> : <SkipButton onClick={ handleNext } /> }
```

Comment:
We can use `isModal` flag to hide the Continue/Skip button when opening Domains page as a modal (from top bar or Plans grid). Here are some reasons to do this:
- _Continue_ is not really the correct wording because you'll go back to the previous screen.
- To be consistent with the Plans modal.
- Go back and Continue/Skip buttons are doing exactly the same thing when `isModal` is `true`.

Reply:
Done. https://github.com/Automattic/wp-calypso/pull/44132/commits/528aadad86eced95b49855eb625852602d5ba236